examples/v2/chat2.nim

```diff
@@ -378,13 +378,14 @@ proc processInput(rfd: AsyncFD, rng: ref BrHmacDrbgContext) {.async.} =
                   symKey: generateSymKey(conf.contentTopic))
 
   if conf.staticnodes.len > 0:
+    echo "Connecting to static peers..."
     await connectToNodes(chat, conf.staticnodes)
   
   var dnsDiscoveryUrl = none(string)
 
   if conf.fleet != Fleet.none:
```

Comment:
Shouldn't it be inside an `elif` to split the two execution branches?
```suggestion
  elif conf.fleet != Fleet.none:
```

Reply:
No strong opinion here, but I think it should be valid to have both static nodes and fleet option configured.

Reply:
@staheri14 I thought the same, but did not change the control flow to keep my change minimal. Imho, too, this behaviour would be more intuitive.
@jm-clius I agree, but it might also be counter-intuitive to connect to the fleet automatically when specifying static nodes.
For me, it would be more natural to have fleet=node as a default if I specify static node.
Also, no strong opinion on my side here.

Reply:
On the other hand, It might be better to leave the control flow as is as some scripts might assume this logic...